### DISCUSSION TITLE
Importing globals.css twice by accident in Next app

### DISCUSSION THREAD

**Discussion by BMCwebdev** [facebook/stylex#692] [General]
We had an issue where, when using npm run dev, the app would compile with StyleX styles. But if you refreshed the page, or typed in a sub-url and went to it, all the StyleX styles would go away.

While digging through our code I found that we were importing `globals.css` twice. Once in the top layout.tsx file (that has things like <html><body> etc). Then again in another file (projectLayout.tsx). Because this file is special to Next, my guess is that when it hit that 2nd import it was over-writing the first one where StyleX styles had been compiled it.

**Comment by nmn** [comment]
Your intuition is on the right track, but usually importing `globals.css` is not a problem. The problem is more that the hot-reloading experience with Next.js and StyleX needs a lot of improvements.
